test/test.js

```diff
@@ -107,9 +107,13 @@ describe('YeldaChat', () => {
     })
 
     describe('yeldaChat.createContainer', async() => {
```

Comment:
Since our async code is running inside the before hook, we don't need the async keywords in the describe function calls anymore.

You can add an eslint rule to make it easier to see: `"require-await": "error"`.